src/components/related_items_outfit/price.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Price({ price, salePrice }) {
+  // const showSalePrice = salePrice === 0;
+
+  return (
+    <span
+      style={{
+        display: 'flex',
+      }}
+    >
+      {salePrice === 0 ? (
+        <p>${price}</p>
+      ) : (
+        <>
+          <p
+            style={{
+              textDecoration: 'line-through',
+            }}
+          >
+            ${price}
+          </p>
+          <p>${salePrice}</p>
+        </>
+      )}
+    </span>
+  );
+}
+
+Price.propTypes = {
+  price: PropTypes.number.isRequired,
+  salePrice: PropTypes.number,
+};
+
+Price.defaultProps = {
+  salePrice: 0,
+};
```

Comment:
Just as we discussed on the zoom, feel free to import the prices component from shared (after I move it over there).